webpack/assets/javascripts/react_app/components/CPUCoresInput/CPUCoresInput.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { WarningTriangleIcon, ErrorCircleOIcon } from '@patternfly/react-icons';
+import { HelpBlock, FormGroup, Grid, Col, Row } from 'patternfly-react';
+import { translate as __ } from '../../common/I18n';
+import NumericInput from '../common/forms/NumericInput';
+import './cpuCoresInput.scss';
+import { noop } from '../../common/helpers';
+
+const CPUCoresInput = ({
+  label,
+  recommendedMaxValue,
+  maxValue,
+  defaultValue,
+  onChange,
+  minValue,
+}) => {
+  const [value, setValue] = useState(defaultValue);
```

Comment:
We should reset the `value` when the consumer is resetting the `defaultValue`.
You can do that with the `useEffect` hook.

Reply:
I added it, I hope I understood well how to use it. Let me know if not.